psalm-baseline.xml

```diff
@@ -1,3 +1,129 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<files psalm-version="5.15.0@5c774aca4746caf3d239d9c8cadb9f882ca29352">
+<files psalm-version="5.20.0@3f284e96c9d9be6fe6b15c79416e1d1903dcfef4">
+  <file src="src/Builder/Encoder/AbstractExpressionEncoder.php">
+    <MixedAssignment>
```

Comment:
A lot of `MixedAssignment` calls are the result of incomplete type annotations throughout the generated files.